### PR TITLE
Slack move to OAuth

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

Updating the Slack integration instructions to reflect new workflows and breaking it out as an individual page. 
https://dbtlabs.atlassian.net/browse/ENTERPRISE-1243

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

File: website/docs/docs/cloud/slack-integration.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+:::
+
+### Prerequisites 
+- You must be an administrator of the Slack workspace. 
+- You must be an account admin to configure Slack notifications in dbt Cloud. For more details, refer to [Users and licenses](/docs/cloud/manage-access/seats-and-users).
+- The integration only supports _public_ channels in the Slack workspace. 
+
+### Set up the Slack integration
+
+1. From the gear menu, select **Account settings** and select **Integrations** from the left navigation pane.
```
Comment (matthewshaver) [inline website/docs/docs/cloud/slack-integration.md]:
```suggestion
1. From the gear menu, select **Account settings** and then select **Integrations** from the left navigation pane. 
```

File: website/docs/guides/zapier-slack.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ recently_updated: true
 
 ## Introduction
 
-This guide will show you how to set up an integration between dbt Cloud jobs and Slack using [dbt Cloud webhooks](/docs/deploy/webhooks) and Zapier. It builds on the native [native Slack integration](/docs/deploy/job-notifications#slack-notifications) by attaching error message details of models and tests in a thread. 
+This guide will show you how to set up an integration between dbt Cloud jobs and Slack using [dbt Cloud webhooks](/docs/deploy/webhooks) and Zapier. It builds on the native [native Slack integration](/docs/cloud/slack-integration) by attaching error message details of models and tests in a thread.
```
Comment (matthewshaver) [inline website/docs/guides/zapier-slack.md]:
```suggestion
This guide will show you how to set up an integration between dbt Cloud jobs and Slack using [dbt Cloud webhooks](/docs/deploy/webhooks) and Zapier. It builds on the native [native Slack integration](/docs/deploy/job-notifications#slack-notifications) by attaching error message details of models and tests in a thread. 
```

File: website/docs/guides/zapier-ms-teams.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ recently_updated: true
 
 ## Introduction 
 
-This guide will show you how to set up an integration between dbt Cloud jobs and Microsoft Teams using [dbt Cloud Webhooks](/docs/deploy/webhooks) and Zapier, similar to the [native Slack integration](/docs/deploy/job-notifications#slack-notifications). 
+This guide will show you how to set up an integration between dbt Cloud jobs and Microsoft Teams using [dbt Cloud Webhooks](/docs/deploy/webhooks) and Zapier, similar to the [native Slack integration](/docs/cloud/slack-integration).
```
Comment (matthewshaver) [inline website/docs/guides/zapier-ms-teams.md]:
```suggestion
This guide will show you how to set up an integration between dbt Cloud jobs and Microsoft Teams using [dbt Cloud Webhooks](/docs/deploy/webhooks) and Zapier, similar to the [native Slack integration](/docs/deploy/job-notifications#slack-notifications). 
```

File: website/docs/docs/deploy/job-notifications.md [hunk 4]
```diff
@@ -80,5 +96,5 @@ If there has been a change in user roles or Slack permissions where you no longe
...
 
-1. From the gear menu, select **Profile settings**. On your **User profile** page, scroll to the **Linked accounts** section.
-1. Find the Slack application in the **Linked accounts** section, click the trash can icon, and click **Unlink**. Channels that you configured will no longer receive Slack notifications. Channels that are configured by other account admins will continue to receive Slack notifications if they still have active Slack integrations. To migrate ownership of a Slack channel notification configuration, have another account admin edit the configuration.
\ No newline at end of file
+1. From the gear menu, select **Account settings**. On the **Integrations** page, scroll to the **OAuth** section.
+1. Click the trash can icon to the right of the Slack integration, and click **Unlink**. Channels that you configured will no longer receive Slack notifications. _This is not an account-wide action._ Channels configured by other account admins will continue to receive Slack notifications if they still have active Slack integrations. To migrate ownership of a Slack channel notification configuration, have another account admin edit their configuration.
```
Comment (matthewshaver) [inline website/docs/docs/deploy/job-notifications.md]:
```suggestion
1. Click the trash can icon to the far right of the Slack integration, and click **Unlink**. Channels that you configured will no longer receive Slack notifications. _This is not an account-wide action._ Channels configured by other account admins will continue to receive Slack notifications if they still have active Slack integrations. To migrate ownership of a Slack channel notification configuration, have another account admin edit their configuration.
```

File: website/docs/docs/deploy/job-notifications.md [hunk 5]
```diff
@@ -52,16 +52,28 @@ If there has been a change in user roles or Slack permissions where you no longe
...
 
-1. From the gear menu, select **Profile settings**. On your **User profile** page, scroll to the **Linked accounts** section.
-1. In the **Linked accounts** section, find the Slack application and click **Link**.
+1. From the gear menu, select **Account settings** and then select **Integrations** from the left navigation pane. 
+1. Locate the **OAuth** section with the Slack application and click **Link**.
    <Lightbox src="/img/docs/dbt-cloud/Link-your-Slack-Profile.png" width="75%" title="Link for the Slack app"/>
-1. Allow dbt Labs to access the Slack workspace. If you are a member of multiple workspaces, you can select the appropriate workspace from the dropdown menu in the upper right corner.
+
+### Logged in to Slack
+If you are already logged in to Slack, the handover will only require allowing the app access. If you are a member of multiple workspaces, you can select the appropriate workspace from the dropdown menu in the upper right corner.
```
Comment (nghi-ly) [inline website/docs/docs/deploy/job-notifications.md]:
```suggestion
If you're already logged in to Slack, the handshake only requires allowing the app access. If you're a member of multiple workspaces, you can select the appropriate workspace from the dropdown menu in the upper right corner.
```

File: website/docs/docs/deploy/job-notifications.md [hunk 6]
```diff
@@ -52,16 +52,28 @@ If there has been a change in user roles or Slack permissions where you no longe
...
    <Lightbox src="/img/docs/dbt-cloud/Allow-dbt-to-access-slack.png" width="75%" title="Allow dbt access to Slack"/>
 
+## Logged out
+
+If you are logged out or the Slack app/website is closed, you will need to authenticate before completing the integraion.
```
Comment (nghi-ly) [inline website/docs/docs/deploy/job-notifications.md]:
```suggestion
If you're logged out or the Slack app/website is closed, you must authenticate before completing the integration.
```

File: website/docs/docs/deploy/job-notifications.md [hunk 7]
```diff
@@ -80,5 +92,5 @@ If there has been a change in user roles or Slack permissions where you no longe
...
 
-1. From the gear menu, select **Profile settings**. On your **User profile** page, scroll to the **Linked accounts** section.
-1. Find the Slack application in the **Linked accounts** section, click the trash can icon, and click **Unlink**. Channels that you configured will no longer receive Slack notifications. Channels that are configured by other account admins will continue to receive Slack notifications if they still have active Slack integrations. To migrate ownership of a Slack channel notification configuration, have another account admin edit the configuration.
\ No newline at end of file
+1. From the gear menu, select **Account settings**. On the **Integrations** page, scroll to the **OAuth** section.
+1. Click the trash can icon to the far right of the Slack integration, and click **Unlink**. Channels that you configured will no longer receive Slack notifications. _This is not an account-wide action._ Channels configured by other account admins will continue to receive Slack notifications if they still have active Slack integrations. To migrate ownership of a Slack channel notification configuration, have another account admin edit their configuration.
```
Comment (nghi-ly) [inline website/docs/docs/deploy/job-notifications.md]:
```suggestion
1. Click the trash can icon (on the far right of the Slack integration) and click **Unlink**. Channels that you configured will no longer receive Slack notifications. _This is not an account-wide action._ Channels configured by other account admins will continue to receive Slack notifications if they still have active Slack integrations. To migrate ownership of a Slack channel notification configuration, have another account admin edit their configuration.
```

File: website/docs/docs/deploy/job-notifications.md [hunk 8]
```diff
@@ -52,16 +52,28 @@ If there has been a change in user roles or Slack permissions where you no longe
...
 - You must be an administrator of the Slack workspace. 
 - You must be an account admin to configure Slack notifications in dbt Cloud. For more details, refer to [Users and licenses](/docs/cloud/manage-access/seats-and-users).
-- Make sure the notification channel (where you want to receive alerts) is a public channel. The integration only supports public channels in the Slack workspace. 
+- The integration only supports _public_ channels in the Slack workspace. 
 
 ### Set up the Slack integration
 
-1. From the gear menu, select **Profile settings**. On your **User profile** page, scroll to the **Linked accounts** section.
-1. In the **Linked accounts** section, find the Slack application and click **Link**.
+1. From the gear menu, select **Account settings** and then select **Integrations** from the left navigation pane.
```
Comment (nghi-ly) [inline website/docs/docs/deploy/job-notifications.md]: should this be "left sidebar"?

File: website/docs/docs/deploy/job-notifications.md [hunk 9]
```diff
@@ -52,16 +52,28 @@ If there has been a change in user roles or Slack permissions where you no longe
...
-- Make sure the notification channel (where you want to receive alerts) is a public channel. The integration only supports public channels in the Slack workspace. 
+- The integration only supports _public_ channels in the Slack workspace. 
 
 ### Set up the Slack integration
 
-1. From the gear menu, select **Profile settings**. On your **User profile** page, scroll to the **Linked accounts** section.
-1. In the **Linked accounts** section, find the Slack application and click **Link**.
+1. From the gear menu, select **Account settings** and then select **Integrations** from the left navigation pane. 
+1. Locate the **OAuth** section with the Slack application and click **Link**.
    <Lightbox src="/img/docs/dbt-cloud/Link-your-Slack-Profile.png" width="75%" title="Link for the Slack app"/>
```
Comment (nghi-ly) [inline website/docs/docs/deploy/job-notifications.md:61]: non-blocking: maybe as a fast-follow, update the screenshot so the Red Arrow does not cross any text

**Review by nghi-ly — COMMENTED** [review]
lgtm! minor nits

**Review by aaronbini — APPROVED** [review]
Looks good! I think the suggested changes in wording would be good to commit as well

File: website/docs/docs/deploy/job-notifications.md (hunk 8 above)
Comment (matthewshaver) [inline website/docs/docs/deploy/job-notifications.md]:
```suggestion
1. From the gear menu, select **Account settings** and then select **Integrations** from the left sidebar. 
```

File: website/docs/docs/deploy/job-notifications.md (hunk 9 above)
Comment (matthewshaver) [reply website/docs/docs/deploy/job-notifications.md:61]: Replaced it!

File: website/docs/docs/deploy/job-notifications.md (hunk 8 above)
Comment (matthewshaver) [reply website/docs/docs/deploy/job-notifications.md]: Changed!

**Review by nghi-ly — APPROVED** [review]
:shipit:

**Pull request merged**